audit_log_feed.tf

```diff
@@ -12,38 +12,69 @@ resource "google_pubsub_topic" "audit_feed" {
   name  = "${local.prefix}audit-feed"
 }
 
-resource "google_logging_organization_sink" "organization_audit_feed" {
-  count       = (var.relay_audit_log && var.organization_id != "") ? 1 : 0
-  name        = "${local.prefix}audit-feed"
+# Workaround for https://github.com/hashicorp/terraform-provider-google/issues/10811
+# Separate resources for include_children true/false to force recreation on change
+
+resource "google_logging_organization_sink" "organization_audit_feed_with_children" {
+  count       = (var.relay_audit_log && var.organization_id != "" && var.audit_log_include_children) ? 1 : 0
+  name        = "${local.prefix}audit-feed-with-children"
+  description = "Organization level audit logs for Stacklet relay"
+
+  org_id           = var.organization_id
+  filter           = local.audit_filter
+  include_children = true
+
+  destination = "pubsub.googleapis.com/${google_pubsub_topic.audit_feed[0].id}"
+}
+
+resource "google_logging_organization_sink" "organization_audit_feed_without_children" {
+  count       = (var.relay_audit_log && var.organization_id != "" && !var.audit_log_include_children) ? 1 : 0
+  name        = "${local.prefix}audit-feed-without-children"
   description = "Organization level audit logs for Stacklet relay"
```

Comment:
@sontek Not a big deal, but the description for with and without children is identical, might be nice to add "(without children)" to it?